content/bn/kubernetes.md

```diff
@@ -0,0 +1,28 @@
+---
+title: কুবারনেটিস
+status: Feedback Appreciated
+category: প্রযুক্তি
+---
+
+## এটা কি
+
+কুবারনেটস (Kubernetes), প্রায়ই K8s হিসাবে সংক্ষিপ্ত হয়, একটি ওপেন সোর্স কন্টেইনার অর্কেস্ট্রেটর যা আধুনিক পরিকাঠামোতে [কন্টেইনারাইজড](/container/) অ্যাপ্লিকেশনের জীবনচক্রকে স্বয়ংক্রিয় করে। এটি একটি ডেটা সেন্টারের অপারেটিং সিস্টেমের মতো, যা অংশ-বিভাগিত সিস্টেম (Distributed System) জুড়ে চলমান অ্যাপ্লিকেশনগুলিকে পরিচালনা করে (ঠিক যেমন আপনার ল্যাপটপের OS আপনার অ্যাপগুলি পরিচালনা করে) ‍।
+
+কুবারনেটস একটি [ক্লাস্টারের](/cluster/) নোড জুড়ে কোনটেনারের সময়সূচী নির্ধারণ করে। এটি বেশ কয়েকটি পরিকাঠামো নির্মাণ, যাকে কখনও কখনও "প্রিমিটিভস" হিসাবে উল্লেখ করা হয়, যেমন একটি অ্যাপের দৃষ্টান্ত, লোড ব্যালেন্সার, পারসিষটেন্ট স্টোরেজ এবং অন্যান্যগুলিকে এমনভাবে একত্রিত করে যাতে সেগুলি অ্যাপ্লিকেশনগুলিতে তৈরি করা যায়।
+
+কুবারনেটস অটোমেশন এবং এক্সটেনসিবিলিটি সক্ষম করে, ব্যবহারকারীদেরকে পুনর্গঠনযোগ্য ও ঘোষণামূলকভাবে অ্যাপ্লিকেশন স্থাপন করতে দেয়। কুবারনেটস ইকোসিস্টেমের সফ্টওয়্যার প্রডাক্ট এবং প্রজেক্টগুলি কুবারনেটস [এপিআই](/application-programming-interface/) প্রসারিত করার জন্য সেই অটোমেশন এবং এক্সটেনসিবিলিটির সুবিধা নেয়। এটি তাদের কুবারনেটসের অটোমেশনকে উদ্দেশ্যসাধনের জন্য ব্যবহার করে এবং তাদের তুলসগুলিকে অভিজ্ঞ কুবারনেটস অনুশীলনকারীদের কাছে আরও অ্যাক্সেসযোগ্য করে তুলতে সক্ষম করে।
+
+## এটা যেসব সমস্যাতে দৃষ্টিপাত করে
+
+ইনফ্রাস্ট্রাকচার অটোমেশন এবং ঘোষণামূলক কনফিগারেশন ম্যানেজমেন্ট দীর্ঘকাল ধরে একটি গুরুত্বপূর্ণ ধারণা এবং [ক্লাউড কম্পিউটিং](/cloud-computing/) জনপ্রিয়তা অর্জন করায় এটি আরও বেশি চাপে পড়েছে।
+
+কম্পিউট রিসোর্সের চাহিদা বৃদ্ধি পাওয়ায় এবং কম প্রকৌশলীর সাথে আরও পরিচালন ক্ষমতা প্রদানের জন্য সংস্থাগুলি চাপ অনুভব করে, এটি পূরণের জন্য নতুন প্রযুক্তি এবং কাজের পদ্ধতির প্রয়োজন হয়।
+
+উপরন্তু, ক্লাউড কম্পিউটিং-এর উত্থান কন্টেনারাইজেশনের সাথে যুক্ত হয়েছিল এবং যে সংস্থাগুলি আরও ঐতিহ্যগত ইনফ্রাস্ট্রাকচার অটোমেশন করতে ব্যস্ত ছিল তাদের কনফিগারেশন অটোমেশন করার জন্য একটি প্রক্রিয়ার প্রয়োজন হয়।
+
+## এটা কিভাবে সাহায্য করে
+
+কুবারনেটস অটোমেশন ঐতিহ্যগত ইনফ্রাসত্রাক্ট এস কোডের মতো সাহায্য করে কিন্তু ভার্চুয়াল বা ফিজিক্যাল মেশিনের তুলনায় কনফিগারেশন ড্রিফ্টকে বেশি প্রতিরোধী কন্টেইনারগুলির সাথে কাজ করার সুবিধা রয়েছে।
+কুবারনেটস ঘোষণামূলকভাবে কাজ করে, যার মানে হল যে অপারেটররা কীভাবে কিছু করতে হবে সে সম্পর্কে নির্দেশনা প্রদান করার পরিবর্তে তারা সাধারণত ম্যানিফেস্ট ফাইল (যেমন YAML) হিসাবে বর্ণনা করে, তারা যা করতে চায়; কুবারনেটস নিজে থেকেই "কিভাবে" করতে হবে তার যত্ন নেয়। এর ফলে কুবারনেটস ইনফ্রাসত্রাক্ট এস কোডের সাথে অত্যন্ত সামঞ্জস্যপূর্ণ।
+
+কুবারনেটস নিজেও নিরাময়  নিজের করে। এর মানে হল যে এটি নিশ্চিত করে যে ক্লাস্টারের প্রকৃত অবস্থা সর্বদা অপারেটরের পছন্দসই অবস্থার সাথে মেলে। কুবারনেটস একটি বিচ্যুতি সনাক্ত করলে, একটি কুবারনেটস কন্ট্রোলার কাজে নামে এবং এটি ঠিক করে। সুতরাং এটি যে ইনফ্রাসত্রাক্ট ব্যবহার করে তা ক্রমাগত কুবারনেটসকেও পরিবর্তন করতে পারে, এবং স্বয়ংক্রিয়ভাবে পরিবর্তনের সাথে খাপ খাইয়ে নেয় এবং নিশ্চিত করে যে এটি অপারেটরের পছন্দসই অবস্থার সাথে মেলে।
```

Comment:
I think this sentence is something unclear `কুবারনেটস নিজেও নিরাময় নিজের করে`. Also, kindly correct the spellings for this word `কুবারনেটস`